model/clusters_mgmt/v1/server_config_type.model

```diff
@@ -21,4 +21,7 @@ class ServerConfig {
 
 	// The kubeconfig of the server
 	Kubeconfig String
+
+	// The topology of a provision shard (Optional)
```

Comment:
Not sure why nothing on this file has the '.', but we usually required it. Do you know if there's some reason for this file not to have that for the comments?

Reply:
Ah, true, I've created the file from scratch, and I forgot.

I'll update it